mesonbuild/dependencies/qt.py

```diff
@@ -319,6 +319,9 @@ def __init__(self, name: str, env: 'Environment', kwargs: T.Dict[str, T.Any]):
             if not self._link_with_qtmain(is_debug, libdir):
                 self.is_found = False
 
+    def get_configtool_args_for_variable(self, variable_name: str) -> T.List[str]:
+        return self.config + ['-query', variable_name]
```

Comment:
This either needs to be a protected method (ie, `_get_configtool_args_for_variable(...)`), or it needs to have a stub version that raises `NotImplementedError` in the `Dependency` base class. The only other thing this probably deserves is a release snippet,. Thanks for working on this!